doc/languages-frameworks/python.section.md

```diff
@@ -497,8 +615,8 @@ buildPythonPackage rec {
 }
 ```
 
-It takes an argument `buildPythonPackage`.
-We now call this function using `callPackage` in the definition of our environment
+It takes an argument `buildPythonPackage`. We now call this function using
+`callPackage` in the definition of our environment
```

Comment:
shouldn't we suggest `python3Packages.callPackage` here?

Reply:
not necessarily, I think the example is clear enough with out having to worry about which interpreter package set your using `callPackage` from.